tests/integration/targets/junos_command/tests/netconf_text/no_config.yaml

```
@@ -0,0 +1,19 @@
+---
+- debug: msg="START netconf_text/no_config.yaml on connection={{ ansible_connection
+    }}"
+
+- name: test bad condition
```

Is this really a good name for this test? Again, I'm not sure I have the right handle on what the issue is, but I would like to see something more descriptive like
```suggestion
- name: test command with no reply
```